Common/ML/include/ML/3rdparty/GPUORTFloat16.h

```diff
@@ -547,9 +547,9 @@ GPUdi() uint16_t BFloat16Impl<Derived>::ToUint16Impl(float v) noexcept
       {
         std::memcpy(&result, reinterpret_cast<char*>(&fl) + sizeof(uint16_t), sizeof(uint16_t));
       } else {
-        static_assert(false, "ERROR!!!");
```

Comment:
The #endif was correct, only the static_assert was wrong.

Reply:
Are you sure the endif was correct? Don't we need to close get_msb_half, then apply it and only then return the result?

Reply:
Now I am confused, the current version in this PR is correct. But my patch was also correct. Perhaps the patch was not applied correctly in the beginning?

Reply:
Yes, I think that was the case. Should be ok now. Let's see what the CI says